tensorboard_logger/tensorboard_logger.py

```diff
@@ -128,7 +131,6 @@ def make_valid_tf_name(name):
 
 _default_logger = None  # type: Logger
 
-
```

Comment:
PEP-8 recommends two blank lines between top-level functions or statements, so I'd prefer to leave them as is.